Documentation/index.md

```diff
@@ -1,17 +1,88 @@
-Welcome to the MonoGame game library documentation hub.
+# Welcome to the MonoGame game library documentation hub.
 
 This area of the site contains the documentation on the API of MonoGame as well as how to use it to create great games.
-Note that this is a work in progress so there will be gaps in the documentation coverage. 
+Note that this is a work in progress so there will be gaps in the documentation coverage.
 
 If you cannot find what you need here you can also look at the [Microsoft XNA documentation](https://msdn.microsoft.com/en-us/library/bb200104.aspx). MonoGame is API compatible
-with [XNA](https://msdn.microsoft.com/en-us/library/bb203940.aspx) even down to the namespaces. So usually what works for XNA will work for MonoGame too. 
+with [XNA](https://msdn.microsoft.com/en-us/library/bb203940.aspx) even down to the namespaces. So usually what works for XNA will work for MonoGame too.
 
-Note that this documentation hub is built from the source code on every commit to the development branch. As such 
+> Note that this documentation hub is built from the source code on every commit to the development branch. As such
 it applies to the development builds available on the [Downloads](http://www.monogame.net/downloads) page. This may include new features which may
-not be available in the current stable release 
+not be available in the current stable release.
 
-### We Need Your Help!
+---
```

Comment:
Lets remove this... it is ugly.

Reply:
:D

Reply:
The entire section or just the header @tomspilman ?

Reply:
Just meant the `---` ... they don't make anything more readable.  Use newlines if we need to space things out more.